### DISCUSSION TITLE
Is it possible to change the home logo to a custom logo?

### DISCUSSION THREAD

**Discussion by lorcanaya** [Azure/ipam#267] [Q&A]
Would like to add a custom logo

**Comment by DCMattyG** [comment]
Hi @lorcanaya, at this point in time it is not. Many of the images in the UI are embedded SVG's which will take some time to adjust the way these are today.

Are you just looking to add a custom on the top bar, or replace all of the images with custom branding?

**Reply by lorcanaya** [reply]
Hey! Not all of the logos. Just the one on the main home page above "Welcome to Azure IPAM". Of course changing the favicon would be a nice bonus.

This would be a huge win for our team.

**Reply by lorcanaya** [reply]
@DCMattyG

**Reply by DCMattyG** [reply]
Are you referring to this one @lorcanaya?

![image](https://github.com/Azure/ipam/assets/12955732/c02bd3c1-6aa7-4a11-a943-05caf0373d1d)

**Reply by lorcanaya** [reply]
![image](https://github.com/Azure/ipam/assets/167252284/bfb043b2-75ef-4299-9789-83aa365e28b7)

This image.
